front-end/src/app/dashboard/dashboard.component.ts

```diff
@@ -1,8 +1,38 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { UserLoginData } from 'app/shared/models/user.model';
+import { userLoggedInAction } from 'app/store/login.actions';
+import { environment } from 'environments/environment';
+import { CookieService } from 'ngx-cookie-service';
 
 @Component({
   selector: 'app-dashboard',
   templateUrl: './dashboard.component.html',
   // styleUrls: ['./dashboard.component.scss'],
 })
-export class DashboardComponent {}
+export class DashboardComponent implements OnInit {
+  constructor(
+    private store: Store,
+    private cookieService: CookieService
+  ) { }
+
+  ngOnInit(): void {
+    this.dispatchUserLoggedInFromCookies();
+  }
+
+  dispatchUserLoggedInFromCookies() {
+    if (this.cookieService.check(environment.ffapiCommitteeIdCookieName)) {
+      const userLoginData: UserLoginData = {
+        committee_id: this.cookieService.get(
+          environment.ffapiCommitteeIdCookieName),
+        email: this.cookieService.get(
+          environment.ffapiEmailCookieName),
+        is_allowed: true,
+        token: null
+      }
+      this.cookieService.delete(environment.ffapiCommitteeIdCookieName)
+      this.cookieService.delete(environment.ffapiEmailCookieName)
+      this.store.dispatch(userLoggedInAction({ payload: userLoginData }));
+    }
+  }
+}
```

Comment:
Can we drop the use of cookies and use the ngrx store 100% or is the use of cookies required?

Reply:
@mjtravers well, the problem was that the server needs to send this data down to the client alongside a redirect.  Because I couldn't use a JSON body in the redirect, I felt my only option was to pass them using cookies.  This code then deletes the cookies once they are received following login.

Reply:
It actually does ultimately store the cookie data in the store UserLoginData object, but only receives it from the server via cookies.